exporter/opentelemetry-exporter-otlp/README.rst

```diff
@@ -1,5 +1,5 @@
-OpenTelemetry Collector Exporter
-================================
+OpenTelemetry Collector Exporters
+=================================
```

Comment:
We should add a comment here that says that this simply pulls in all available opentelemetry otlp exporters.

Reply:
Done